### ISSUE TITLE
XXX法 is it "Act" or "Law"?

### ISSUE THREAD

**Issue by JMdictProject** [JMdictProject/JMdictIssues#35]
I have noticed that we are rather inconsistent in our translations of the titles of legislation passed by the Japanese parliament. About half the time we say "XXX Law" and for the rest we say "XXX Act". It would be good to be consistent.

For me the correct and formal term is "XXX Act"; it's the usual term used in Australia, the UK, New Zealand, the US, etc. You sometimes hear "XXX Law" but it sounds informal or colloquial.

Looking at Japanese sources, I see:
(a) the JE dictionaries seem to always say "XXX Law" (GG5, 中辞典, etc.) when referring to Japanese legislation, and "Act" when referring to the US, UK, etc.
(b) the government's "Standard Legal Terms Dictionary" doesn't include the names of legislation, but uses "Act" as the generic term.
(c) the government "Japanese Law Translation" site (e.g. http://www.japaneselawtranslation.go.jp/law/detail/?vm=04&re=01&id=94) uses "Act", e.g. for 予防接種法 it has "Immunization Act" - we currently have "Preventive Vaccination Law".

Should we settle on one, and if so which?

**Comment by robinjmdict** [comment]
I prefer "Act". If the Japanese Law Translation site is using "Act", it makes sense for us to use it as well.

**Comment by Marcusjmdict** [comment]
I prefer using what's in japaneselawtranslation.go.jp to what's in GG5.

**Comment by Marcusjmdict** [comment]
Related - Japan has around 1900 individual laws. I think a large percentage of them appear in at least one of the major dictionaries (koj, daij, gg5, nikk, etc). For the ones that have really long names that are noun phrases (xxxをyyyする法律 etc.), they usually only appear in abbreviated form in the same refs (these abbreviations don't actually appear in the title or the text of the laws themselves but apparently have become established expressions through other means). I've been self-approving new law submissions whenever they appear in at least one of these sources, as the official English translations are readily accessible on japaneselawtranslations.go.jp, but could we decide that at least all of them that aren't long phrases are worth including even if they're not in any of the major refs? (and that the abbr forms of the phrase ones are worth having when they appear in one of the major dictionaries). 

(I don't particularly object to having the phrase ones as well, personally)

Might it also be possible to automate the addition of them? Weblio seems to have a wa-ei legal dictionary sourced from japaneselawtranslations.go.jp.

**Comment by Marcusjmdict** [comment]
In the [アイヌ新法](https://www.edrdg.org/jmdictdb/cgi-bin/entr.py?svc=jmdict&sid=&e=2174591) entry, Robin has suggested we do not use the full, formal English translations that appear on Japaneselawtranslations.go.jp for entries on shorter "popular names" (通称) of laws, drop the [abbr] tags and the "abbr. of [full name of the law in Japanese]" notes on the few that have them and "If needed, include a short description of the law. If the name is used to refer to more than one law, list the years they were enacted in brackets."

The short names in those entries we already have are usually whatever GG5 has chosen to translate them as, and they are usually formatted as if they were the formal name, with capitals, e.g. アイヌ新法 "New Ainu Law". I don't think this is ideal. The two hits on The Japan Times for this term are for "the new Ainu law." (in other articles, they refer to it as the "Ainu Culture Promotion Act" or "(an) Ainu culture promotion law")

For another example, we have  [不正アクセス禁止法](https://www.edrdg.org/jmdictdb/cgi-bin/entr.py?svc=jmdict&sid=&q=2828793.1) as "Unauthorized Computer Access Law (2000); Law Banning Illegal Access". The full Japanese name of the law is 不正アクセス行為の禁止等に関する法律 and the [official English translation](http://www.japaneselawtranslation.go.jp/law/detail/?id=3933&vm=&re=) is "Act on Prohibition of Unauthorized Computer Access" - not really much longer than our (or rather GG5's) abbreviated form. Is it really better to use what's likely a GG5-ism than the formal name here, and give an impression that it's actually the formal name?

On the note of tagging shorter names of laws as [abbr], Robin said "I also don't think the abbr tag is needed here. アイヌ新法 is a separate term. There's no "新" in the full name." and while this is true for アイヌ新法, in the case of 不正アクセス禁止法, all the characters appearing in the shorter form appears in the full form. Should we call that one an abbr, but not the other? I think this ends up being a little messy, and coupled with the removal of the "abbr. of XXX" notes we are in no way indicating these shortened forms aren't actually the formal names of the laws.

I don't think we should include descriptions in any law entry. I think doing it "when needed" is a very vague guideline. I also don't think we need to list the years different laws known by the same abbreviation were enacted unless we're actually listing them under separate names.

**Comment by Marcusjmdict** [comment]
(it's worth noting that アイヌ新法 is called a 略称 in Nipponica and Chiezo Mini (but "通称" in Daijisen))

**Comment by JMdictProject** [comment]
In general, I don't have a problem with using the "official" translations as the leading gloss. In some cases, they seem a bit clunky and unidiomatic English. It might be good if we could also include an abbreviated title where appropriate.

For things like アイヌ新法, which are an abbreviation or nickname, a shorter gloss is quite appropriate, and there should be an xref to the full title of the Act, if possible.

**Comment by robinjmdict** [comment]
The problem with the long-form names is that they often don't make good translations. "アイヌ新法" wouldn't usually be rendered as "Act on the Promotion of Ainu Culture, and Dissemination and Enlightenment of Knowledge about Ainu Tradition, etc."

I take back my suggestion to drop the abbr tags. The tag is a way of indicating that it's not the official name.

There was a proposal a while back to introduce an "official name" gloss type. Could we include the full name as a second [offic] gloss?

> I also don't think we need to list the years different laws known by the same abbreviation were enacted unless we're actually listing them under separate names.

It might be worth including the year the law was enacted for _all_ entries.

> there should be an xref to the full title of the Act, if possible.

I don't think it would be appropriate to add any of the forms that end in 〜に関する法律.

**Comment by Marcusjmdict** [comment]
>The problem with the long-form names is that they often don't make good translations. "アイヌ新法" wouldn't usually be rendered as "Act on the Promotion of Ainu Culture, and Dissemination and Enlightenment of Knowledge about Ainu Tradition, etc."

This is true and the full name of the law might not be an appropriate gloss. I'm wary of making it "New Ainu Law" though, which makes it seem like an official name used in English. I might prefer "the new Ainu law" + an explanation for entries like this one. But for entries like  不正アクセス禁止法 that I mentioned above, where the abbreviated and unabbreviated forms don't differ all that much in content and length, I would prefer using the formal title of the law rather than a GG5-ism, esp. a capitalized one.

> It might be worth including the year the law was enacted for all entries.

I feel like the year a law was enacted isn't really all that important information. It's not as if the year it was enacted has any bearing on the law's validity. Adding the year to all of our current and future ~法 entries seems like a lot of work for little benefit.

>For things like アイヌ新法, which are an abbreviation or nickname, a shorter gloss is quite appropriate, and there should be an xref to the full title of the Act, if possible.

In general these abbreviations are for long-ish titles, e.g. 〜に関する法律. So if we want to mention what it's an abbreviation of, which I think we might as well, then having a note on the top of the entry seems the best way of doing it, even if it can end up slightly unsightly (e.g. "abbr. of アイヌ文化の振興並びにアイヌの伝統等に関する知識の普及及び啓発に関する法律").

**Comment by robinjmdict** [comment]
> I'm wary of making it "New Ainu Law" though, which makes it seem like an official name used in English.

I don't think it matters. The abbreviated names aren't official even in Japanese, but they're still names (used as headwords in the kokugos and encyclopedias) and should therefore be capitalised.

On Reverso,  "暴力団対策法" is translated as "Anti-Organized Crime Group Law", "Gang Countermeasure Law" and "Anti-Boryokudan Act". The Japanese Law Translation site's translation of the full name  is "Act on Prevention of Unjust Acts by Organized Crime Group Members". I much prefer the shorter glosses. I think we should just pick the one we like the best (or make up our own).

For something like 出資法, whose long-form translation is "Act Regulating the Receipt of Contributions, the Receipt of Deposits, and Interest Rates", we'd probably want to include a little than just "Investment Act" (or whatever translation we decided to go with).
